docs/screenshot-tests.md

```diff
@@ -42,25 +41,58 @@ Option | Description
 `--cap-add=SYS_ADMIN` | Run container with system admin privileges
 `-e MDC_...="${MDC_...}"` | Set MDC_GCLOUD_SERVICE_ACCOUNT_KEY env variable
 
-#### STEP 3: Run Tests
+### STEP 3: Run Tests
 
-At this point you are ready to run. This Docker container is mainly designed for running the screenshot tests. The container already has 1) cloned the repo and 2) `npm install` the project dependencies.
+This Docker container is mainly designed for running the screenshot tests. The container already has 1) cloned the repo and 2) `npm install` the project dependencies.
 
-The next thing you need to do is start the server and run the tests.
+In the Docker container, start the server:
 
 ```
-git checkout <INSERT_YOUR_BRANCH_NAME>
+git pull
+git checkout --track origin/<BRANCH_NAME>
```

Comment:
i don't think you need the --track. After you pull you can just do `git checkout <BRANCH_NAME>`